### PR TITLE
chore(add): lock app button

### LINKED ISSUE #5326: App Lock Button
User wanted a way to get back to the lock page, here it is

### PR DESCRIPTION
**Pull request by JustZacca**
### What this PR does 📖
- Add app lock button

### Which issue(s) this PR fixes 🔨
- Resolve #5326 

### Special notes for reviewers 🗒️
- 

### Additional comments 🎤
-

### CODE REVIEW

File: components/interactables/QuickProfile/QuickProfile.vue [hunk 1]
```diff
@@ -154,5 +161,12 @@ function openSettings() {
 function setMenuVis(val: boolean) {
   isStatusMenuVisible.value = val
 }
+
+async function toggleLock() {
+  window.console.log('pressdsafads')
```
Comment (stavares843) [inline components/interactables/QuickProfile/QuickProfile.vue:166]: can we ditch this? 🔨

Comment (JustZacca) [reply components/interactables/QuickProfile/QuickProfile.vue:166]: yeah, I'm dumb, I forgot to delete it

**Comment by stavares843** [comment]
https://user-images.githubusercontent.com/29093946/196780180-b4221574-74ab-4cb4-a004-692fe95d7a06.mov

neat 🎉

File: components/interactables/QuickProfile/QuickProfile.vue [hunk 2]
```diff
@@ -47,6 +47,13 @@
     >
       <arrow-right-icon size="1x" />
     </InteractablesInput>
+    <TypographyText color="white">
+      <InteractablesButton
+        color="white"
+        :text="$t('popups.quick_commands.title')"
+        @click="toggleLock"
+      />
+    </TypographyText>
```
Comment (josephmcg) [inline components/interactables/QuickProfile/QuickProfile.vue:56]: could you conditionally render this for only your own quick profile? There should be a computed that you can use

you can remove the typographyText around the button. Also double check the color prop options, white isn't a valid color in the type declaration. I think you want 'dark'

File: components/interactables/QuickProfile/QuickProfile.vue [hunk 3]
```diff
@@ -154,5 +161,11 @@ function openSettings() {
 function setMenuVis(val: boolean) {
   isStatusMenuVisible.value = val
 }
+
+async function toggleLock() {
+  $store.commit('accounts/lock')
+  location.reload()
+  close()
+}
```
Comment (josephmcg) [inline components/interactables/QuickProfile/QuickProfile.vue:169]: doesnt need to be async

**Review by josephmcg — CHANGES_REQUESTED** [review]

**Comment by josephmcg** [comment]
I adjusted store so this is only rendered in the bottom left quick profile since there are several ways to toggle your own quick profile (message name click, right sidebar click).

**Review by josephmcg — APPROVED** [review]

**Pull request merged**